Generator/GeneratorRegistryInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Pim\Bundle\DataGeneratorBundle\Generator;
+
+/**
+ * Interface for Generators registries.
+ *
+ * @author    Pierre Allard <pierre.allard@akeneo.com>
+ * @copyright 2016 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+interface GeneratorRegistryInterface
```

Comment:
The good solution will be to create our own `ServiceRegistry` component like Sylius. You have several registries with different signatures...